dev/io.openliberty.microprofile.openapi.ui.internal_fat/fat/src/io/openliberty/microprofile/openapi/ui/internal/fat/tests/UIBasicTest.java

```diff
@@ -101,6 +101,10 @@ public void testHttpUI(){
 
     @Test
     public void testHttpsUI(){
+        //Reduce possibility that Server is not listening on its HTTPS Port
+        //Especially for Windows if certificates are slow to create
+        server.waitForStringInLog("CWWKO0219I: TCP Channel defaultHttpEndpoint-ssl", 60000);
```

Comment:
We don't need to set a manual timeout here. The standard timeout is two minutes to account for slow build machines and I don't think we need to reduce it for these tests.

In general I would use `CWWKO0219I:.*defaultHttpEndpoint-ssl` as the string to search for to avoid relying on the server running with the English locale, and have a comment with an example of the log line we're looking for.